dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -158,8 +160,11 @@ class DdAndroidGradlePlugin : Plugin<Project> {
                         SdkCheckLevel.WARN -> {
                             LOGGER.warn(MISSING_DD_SDK_MESSAGE.format(variant.name))
                         }
-                        SdkCheckLevel.NONE -> {
-                            // no-op, ignore that and make compiler happy
+                        else -> {
```

Comment:
Has the `SdkCheckLevel.NONE` value been removed ?

Reply:
No, check for `.NONE` was moved out of lambda, before action creation. So it cannot be here logically, but it should be some branch to make compiler happy.